packages/f-metadata/src/BrazeDispatcher.js

```diff
@@ -0,0 +1,255 @@
+import ContentCards from './services/contentCard.service';
+import isAppboyInitialised from './utils/isAppboyInitialised';
+
+/* braze event handler callbacks */
+
+/**
+ * Dispatches events for in-app message clicks to the registered handlers
+ * @param message
+ * @this BrazeDispatcher
+ */
+function interceptInAppMessageClickEventsHandler (message) {
+    this.inAppMessageClickEventCallbacks.forEach(callback => callback(message));
+}
+
+/**
+ *
+ * @param message
+ * @this BrazeDispatcher
+ */
+function interceptInAppMessagesHandler (message) {
+    if (message instanceof this.appboy.ab.InAppMessage) {
+        /**
+         * Always subscribe click action to second button
+         * as this is always "success" as opposed to "dismiss"
+         * as confirmed with CRM (AS)
+         */
+        this.inAppMessagesCallbacks.forEach(callback => callback(message));
+        if (message.buttons && message.buttons.length >= 2) {
+            const button = message.buttons[1]; // eslint-disable-line prefer-destructuring
```

Comment:
Could use destructuring so that you can remove the ignore comment?

```suggestion
            const [,button] = message.buttons;
```